lacommunaute/templates/forum/forum_documentation_category.html

```diff
@@ -11,6 +11,9 @@
                     {% endif %}
                     <div class="card-body pb-0">
                         <p class="h3 lh-base">{{ node.obj.name }}</p>
+                        {% for tag in node.obj.tags.all %}
```

Comment:
je rajouterais bien au dessus du `{% for tag in node.obj.tags.all %}`, un `{% if tags %}` avec un `<div>` dedans pour enrober les `<span.tag>` (car ils sont en display:inline-block)